crates/http-trigger/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "spin-http-trigger"
```

Comment:
Bike-shedding, I prefer `spin-trigger-http` for two reasons:
- it matches the pattern of external trigger plugins, e.g. `spin-trigger-sqs`
- it shows the relationship with `spin-trigger` (and puts them next to each other in my editor file tree :sweat_smile:)